digital-objects/ref-organ/lymph-node-female-right/v1.1/raw/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+title: 3D Reference Organ for Lymph Node, Female, Right
+description:
+  "[This reference organ](https://humanatlas.io/3d-reference-library) was
+  created using the Clearing-enhanced 3D (Ce3D) method developed by Dr. Weizhe Li,
+  Laboratory of Immune System Biology, National Institute of Allergy and Infectious
+  Diseases, NIH using data documented in https://www.pnas.org/content/114/35/E7321.
+  While the size and cellular composition of mouse and human lymph nodes varies between
+  species, the overall anatomy is well-conserved."
+creators:
+  - fullName: Li Yao
+    firstName: Li
+    lastName: Yao
+    orcid: 0000-0003-2390-083X
+project_leads:
+  - fullName: "Katy B\xF6rner"
```

Comment:
Katy's name is mangled for all these metadata.yaml files. Please fix. See https://github.com/hubmapconsortium/hra-kg/blob/main/digital-objects/ref-organ/lymph-node-female-right/v1.0/raw/metadata.yaml#L16 for what it should look like.